e2e/tests/validator.go

```diff
@@ -10,8 +10,12 @@ import (
 	"github.com/pokt-network/pocket/runtime/defaults"
 )
 
-// rpcURL of the pod that the test harness drives
-var rpcURL string
+var (
+	// rpcURL of the pod that the test harness drives
+	rpcURL string
+	// targetPod is the name of pod used for E2E tests
+	targetPod string = "deploy/pocket-v1-cli-client"
```

Comment:
Do you think it would be better to have `"deploy/dev-cli-client"` here and just rename the deployment on LocalNet? 

https://github.com/pokt-network/pocket/blob/main/build/localnet/manifests/cli-client.yaml#L4

Reply:
@dylanlott I would change the name to be the same across DevNet and LocalNet. I think this removes the confusion about names of the resources in the future. Here are a couple of files that also need a change:

https://github.com/pokt-network/pocket/blob/b4a039741460ac41671cebf81756ea33100de5d1/build/localnet/Tiltfile#L158
https://github.com/pokt-network/pocket/blob/b4a039741460ac41671cebf81756ea33100de5d1/Makefile#L501

Lmk if you don't have bandwidth to do that, I'm happy to take care of that.

Reply:
That works for me! I will make those changes and change the name of this PR then 👍